Source/Core/DolphinQt/NetPlay/NetPlayBrowser.cpp

```diff
@@ -305,9 +305,13 @@ void NetPlayBrowser::accept()
 
     SetQWidgetWindowDecorations(dialog);
     if (dialog->exec() != QDialog::Accepted)
+    {
+      delete dialog;
       return;
+    }
 
     const std::string password = dialog->textValue().toStdString();
+    delete dialog;
```

Comment:
Could we use `unique_ptr` instead so we don't have to remember to manually delete?

Reply:
I think it could just be on the stack instead of the heap, but I wasn't 100% sure so I went with this instead.

Reply:
It seems to work fine, so I'll go with that option.